app/Controller/Component/RoleComponent.php

```diff
@@ -65,6 +65,60 @@ public function approverFor($coPersonId) {
     // find() will return id => id with only one field specified, so just pull the keys
     return array_keys($efs);
   }
+
+  /**
+   * Determine what CO Enrollment Flows a CO Person may approve by being a member of
+   * the special approvers groups
+   *
+   * @since  COmanage Registry v0.8.3
+   * @param  Integer CO Person ID
+   * @return Array List CO Enrollment Flow IDs
+   * @throws InvalidArgumentException
+   */
+  public function approverForEof($coPersonId, $cou_level=false) {
```

Comment:
Is `Eof` supposed to be `EnrollmentFlow`? If so it should be spelled out for clarity.